app/test_main.py

```diff
@@ -1 +1,34 @@
-# write your code here
+from unittest import mock
+import pytest
+from app.main import can_access_google_page
+
+
+@pytest.mark.parametrize(
+    "url, valid_url, internet_connection, expected",
+    [
+        # Valid URL and internet connection
+        ("https://www.google.com", True, True, "Accessible"),
+        # Invalid URL but internet connection
```

Comment:
don't think you need to parametrize url - it can be anything since you mock function that actually does a request